src/types.ts

```diff
@@ -15,20 +15,36 @@ export interface PlayerType {
   playerIndex: number;
   port: number;
   characterId: number | null;
-  characterColor: number | null;
-  startStocks: number | null;
   type: number | null;
+  startStocks: number | null;
+  characterColor: number | null;
+  teamShade: number | null;
+  handicap: number | null;
   teamId: number | null;
+  staminaMode: boolean | null;
+  silentCharacter: boolean | null;
+  invisible: boolean | null;
+  lowGravity: boolean | null;
+  blackStockIcon: boolean | null;
+  metal: boolean | null;
+  startOnAngelPlatform: boolean | null;
+  rumbleEnabled: boolean | null;
+  cpuLevel: number | null;
+  offenseRatio: number | null;
+  defenseRatio: number | null;
+  modelScale: number | null;
   controllerFix: string | null;
   nametag: string | null;
   displayName: string;
   connectCode: string;
   userId: string;
 }
 
-export enum GameMode {
+export enum GameModeType {
```

Comment:
```suggestion
export enum GameMode {
```
Can we revert this and not make breaking changes where possible? Ideally this doesn't have to be a breaking change.